temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -226,41 +223,24 @@ private ScheduledExecutorService grpcConnectionManager() {
             .build());
   }
 
-  /**
```

Comment:
Why the javadoc is gone? It looks pretty applicable to the new code

Reply:
I've inlined a function and IDE apparently has removed the comment, let me put it back.